src/fe-gtk/gtkutil.c

```diff
@@ -598,6 +599,10 @@ gtkutil_window_new (char *title, char *role, int width, int height, int flags)
 		gtk_window_set_transient_for (GTK_WINDOW (win), GTK_WINDOW (parent_window));
 		gtk_window_set_destroy_with_parent (GTK_WINDOW (win), TRUE);
 	}
+	if (flags & START_ICONIFIED)
```

Comment:
I knew. My pull request try to minimize the changes on the code.

How would you like to fix that? Make the function accept one more parameter?